### ISSUE TITLE
Automatically add scss file and documentation when generating a component

### ISSUE THREAD

**Issue by hdurix** [tikui/tikui-cli#40]
For instance, when creating `buttom` in `src/atom`, is there any reason not to add:
- `include:componentDoc(height=55) button/button.md` in `src/atom/atom.pug`
- `@import 'button/button';` in `src/atom/_atom.scss`
?

This would speed up development and help newbies :)

**Comment by Gnuk** [comment]
I understand but I'm not sure it's a good idea. First of all because with the linter, an empty file is not possible, that's why I used a comment in the tikui.scss file. And also because the CSS is the thing that people will make using the tool but the html is the thing used to document, so, it may not be a problem to have a structure for the documentation but not for the CSS. Don't hesitate to answer this ;)

**Comment by Gnuk** [comment]
Hum I answer on the bad issue for the previous comment. But I think it's not easy to understand the structure the users want to add the componentDoc or templateDoc.

**Comment by hdurix** [comment]
ideas:
- `componentDoc` by default for all then the user can change
- `templateDoc` by default for templates and `componentDoc` for other
- `componentDoc` & `templateDoc` could be merge with a boolean for showing or not the code (could be more explicit for user)
- option on `tikui create` to choose what you want (`componentDoc` or `templateDoc` or `none`)

Otherwise, I think using Tikui for the first time is a bit tricky

**Comment by Gnuk** [comment]
I let you suggest a PR for the simplest option you see (if you have an idea) . But you need to keep in mind that it's not because the generation is using an atomic design structure that people are forced to use this structure (if they don't want/need).

**Comment by hdurix** [comment]
yes even if tikui-cli is already generating an atomic design structure :stuck_out_tongue:
